src/plugins/vis_type_timeseries/server/lib/search_strategies/strategies/abstract_search_strategy.ts

```diff
@@ -66,14 +66,15 @@ export class AbstractSearchStrategy {
     this.additionalParams = additionalParams;
   }
 
-  async search(req: ReqFacade, bodies: any[], options = {}) {
+  async search(req: ReqFacade, bodies: any[], options = {}, dataSourceId: string) {
```

Comment:
dataSouurceId should be optional, right?

Reply:
Yes, this is fixed